### PR TITLE
Complete seeding functions

### PR DESCRIPTION
**Pull request by hcnimi**

### CODE REVIEW

File: src/database/index.js [hunk 1]
```diff
@@ -30,178 +31,197 @@ const User = db.define('users', {
...
-Category.hasMany(Style, {
-  foreignKey: {
-    allowNull: false
-  }
-});
-
+Color.hasMany(Item, { foreignKey: { allowNull: false } });
+Category.hasMany(Item, { foreignKey: { allowNull: false } });
+Item.belongsTo(Category, { allowNull: false });
+Category.hasMany(Style);
```
Comment (skoshkarli) [inline src/database/index.js]: any reason for allowNull: false not being here anymore?

File: src/database/index.js [hunk 2]
```diff
@@ -30,178 +31,197 @@ const User = db.define('users', {
...
-// tables should be sync'd in the following order: category, user, closet, item, calendar, outfit, style, season, outfits_items, styles_seasons
-Category.sync().then(() => User.sync().then(() => Closet.sync().then(()=> Color.sync().then(() => Item.sync().then(() => Calendar.sync().then(() => Outfit.sync().then(() => Style.sync().then(() => Season.sync().then(() => OutfitItem.sync().then(() => StyleSeason.sync()))))))))));
+Outfit.hasMany(Calendar);
+Calendar.belongsTo(Outfit);
+
+/* promise chain is required to ensure tables are created in the correct sequence so that associations can be set up
+ properly. tables should be sync'd in the following order: category, user, closet, item, calendar, outfit, style,
+  season, outfits_items, styles_seasons
+  */
+db.sync()
```
Comment (skoshkarli) [inline src/database/index.js]: async await?

File: src/database/seedData.js [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+      Pants: ['Legging', 'Capris/Cropped', 'Trouser'],
+      Shorts: ['Romper', 'Short']
+    },
+    Men: {
+      Tops: ['Dress Shirt', 'Polo Shirt'],
+      Pants: ['Chino'],
+      Shorts: ['Hybrid', 'Chino', 'Swim']
+    }
+  },
+  // styles: [
```
Comment (skoshkarli) [inline src/database/seedData.js]: should this be commented out?

Comment (hcnimi) [reply src/database/seedData.js]: yes. not being used, but keeping it bc I may need it for seeding.

File: src/database/index.js (hunk 2 above)
Comment (hcnimi) [reply src/database/index.js]: no.

**Pull request merged**